edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Edgelet.Test/ModuleManagementHttpClientTest.cs

```diff
@@ -271,6 +273,33 @@ public async Task ModuleLogsTest(string serverApiVersion, string clientApiVersio
             Assert.Equal(buffer.Length, bytesRead);
         }
 
+        [Theory]
+        [MemberData(nameof(VersionDateTimeMap))]
+        public async Task ModuleLogsTestInvalidDateTime(string serverApiVersion, string clientApiVersion, string since, string until)
+        {
+            // Arrange
+            IModuleManager client = new ModuleManagementHttpClient(this.serverUrl, serverApiVersion, clientApiVersion);
+            // Act and Assert
+            var ex = await Assert.ThrowsAsync<ArgumentException>(async () =>
+            {
+                await client.GetModuleLogs("edgeHub", false, Option.Some<int>(10), Option.Some<string>(since), Option.Some<string>(until), Option.Some(true), CancellationToken.None);
+            });
+            Assert.Contains("The correct format should be yyyy-MM-dd'T'HH:mm:ssZ", ex.Message);
+        }
+
+        [Theory]
+        [MemberData(nameof(VersionMap))]
+        public async Task ModuleLogsValidDateTime(string serverApiVersion, string clientApiVersion)
+        {
+            // Arrange
+            IModuleManager client = new ModuleManagementHttpClient(this.serverUrl, serverApiVersion, clientApiVersion);
+
+            // Act and Assert
+            string since = DateTime.Now.AddDays(-1).ToString("yyyy-MM-dd'T'HH:mm:ssZ");
+            string until = DateTime.Now.AddDays(+1).ToString("yyyy-MM-dd'T'HH:mm:ssZ");
+            await client.GetModuleLogs("edgeHub", false, Option.Some<int>(10), Option.Some<string>(since), Option.Some<string>(until), Option.Some(true), CancellationToken.None);
```

Comment:
Should we be asserting anything or is it enough that this doesn't throw?

Reply:
It is enough for the purposes of this unit test to make sure it doesnt throw